src/components/App.js

```diff
@@ -0,0 +1,42 @@
+import Header from './Header';
+import React, { useState, useEffect } from 'react';
+import { fetchFolderData } from '../api';
+import Banner from './Banner';
+import CardContainer from './CardContainer';
+import 'normalize.css';
+import '../reset.css';
+import Footer from './Footer';
```

Comment:
import를 하는 순서도 한번 정리를 해봐도 좋을 것 같은데요~
아래와 같은 느낌으로 해보면 괜찮을 것 같아요!

여기서는
```suggestion
import React, { useState, useEffect } from 'react';

import Header from './Header';
import Banner from './Banner';
import CardContainer from './CardContainer';
import Footer from './Footer';
// 여기 개행은 취향
import { fetchFolderData } from '../api';

import 'normalize.css';
import '../reset.css';

```

Reply:
제 개인 취향이기는 한데 컴포넌트를 import하는 순서도 저는 실제로 어떤 순서대로 사용하냐에 따라서 맞춰서 해주는 편이에요~

```js
// return에서 Header, Banner, CardContainer, Footer 순서이기 때문에 import도 동일한 순서로
return (
      <Header />
      <Banner name={name} owner={owner} />
      <CardContainer links={links} />
      <Footer />
)
```